packages/manager/modules/backup-agent/src/pages/services/dashboard/agent/_components/AgentActionsCell.component.tsx

```diff
@@ -1,5 +1,7 @@
 import { useId } from 'react';
```

Comment:
Can you rebase on my branch to fix the display action menu : https://github.com/ovh/manager/pull/20599 ?